### ISSUE TITLE
Would it be better to have mitmweb use option "content_view_lines_cutoff" as well?

### ISSUE THREAD

**Issue by sanlengjingvv** [mitmproxy/mitmproxy#5398]
#### Problem Description
There is an option "content_view_lines_cutoff" is effective in `mitmproxy`.
But it is not work in `mitmweb`.
The web project uses a new constant in _web/src/js/components/contentviews/useContent.tsx_
`export const SHOW_MAX_LINES = 80;`

#### Proposal
Would it be better to have mitmweb use option "content_view_lines_cutoff" as well?

**Comment by mhils** [comment]
Yes, that sounds reasonable if it can be implemented without too much new complexity. 👍
